README.md

```diff
@@ -1,117 +1,84 @@
-<p align="center">
-  <a href="https://github.com/actions/typescript-action/actions"><img alt="typescript-action status" src="https://github.com/actions/typescript-action/workflows/build-test/badge.svg"></a>
-</p>
+# GitHub Action - Add Comment Action
 
-# Create a JavaScript Action using TypeScript
+This GitHub action will add a comment to an issue with some minor formatting based on the status
 
-Use this template to bootstrap the creation of a JavaScript action.:rocket:
+<!-- Add test badge once proper tests are added -->
```

Comment:
Should this be logged as an issue in the repo?

Reply:
Yes it should be!  https://github.com/ActionsDesk/add-comment-action/issues/8